travvy/src/components/AttractionDetails.vue

```diff
@@ -1,6 +1,21 @@
 <template>
   <div>
-      <h1>Attraction Details for {{}} </h1>
+    <!-- Button to go to go back to the AttractionsList page --> 
+    <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
+
+    <h1>CN Tower </h1>
+    <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
+    <br>
+    <h1>Description</h1>
+    <p>The CN Tower is a 553.3 m-high concrete communications and observation tower located in Downtown Toronto, Ontario, Canada.</p>
```

Comment:
I can see that youve hard coded this description for this instance, however when we are able to access the data from the database through the backend server, you are planning on changing this to be dynamic, correct?

Reply:
Good catch, Rushan! Yes, that's correct, for the time being, I'm just stubbing for testing purposes. After this iteration, we'd definitely want the data to be pulled directly from our database.

Reply:
Ok great!